usersperhour.json

```diff
@@ -0,0 +1,142 @@
+{  
```

Comment:
Since we're fetching from the API now, we can get rid of this file with the dummy data